jerry-core/ecma/base/ecma-helpers-conversion.c

```diff
@@ -818,12 +818,10 @@ ecma_uint32_to_utf8_string (uint32_t value, /**< value to convert */
  *
  * @return number - result of conversion.
  */
-ecma_number_t
+ecma_number_t __attr_always_inline___
```

Comment:
I agree with @akiss77. If it doesn't have any performance gain, then don't add it.